orca_nw_lib/vlan.py

```diff
@@ -89,6 +87,7 @@ def _create_vlan_db_obj(device_ip: str, vlan_name: str = None):
             if v.name == item.get("name"):
                 members.append(item)
         vlans_obj_vs_mem[v] = members
+        print(members)
```

Comment:
@rahul-1475 , Print can be removed. Use proper logging instead if needed.
```suggestion
        print(members)
```

Reply:
removed print